app/controllers/projects_controller.rb

```diff
@@ -180,8 +180,8 @@ def create
 
     respond_to do |format|
       if @project.save
-        tags = %W[project_id:#{@project.id} user_id:#{current_user.id}]
-        MetricUtil.put_metric_now("projects.created", 1, tags)
```

Comment:
Shouldn't we keep datadog metrics ? 
At least until we can have a dashboard to snowflake.

Reply:
@tfrcarvalho ah good point. Forgot about our dashboard for some reason. Will change

Reply:
@tfrcarvalho updated :)